### DISCUSSION TITLE
kitty black screen

### DISCUSSION THREAD

**Discussion by arun54321** [kovidgoyal/kitty#4594] [Q&A] [answered]
![jjjj](https://user-images.githubusercontent.com/42943340/151690331-38c797e6-3192-4653-8999-5b53837a012f.jpg)

I didn't have any configuration files yet. I'm not sure whether it's a bug or not.

OS: archlinux
v: 0.24.1-1

**Comment by kovidgoyal** [comment]
Its an issue with something lower down in the stack on your system. Either graphics drivers or fontconfig/freetype. You can try running kitty in another terminal with --debug-rendering it might give you more info on the issue.

**Comment by page-down** [comment]
If it's a font issue, try the following steps.

First list the mono fonts, then specify the font when running kitty, and finally generate and edit the config file.

```shell
kitty +list-fonts

kitty --config=NONE -o 'font_family <font name>' --debug-font-fallback

# press ctrl+shift+f2 to edit kitty.conf
```

**Comment by arun54321** [comment, marked as answer]
Looks like the issue is caused by infinality patched freetype2. It works fine now after removing it. I don't understand why kitty has issues with infinality freetype.

**Reply by Ethanshibu** [reply]
i have this issue after installing arch with i3 on a virtualbox using an archinstall script. could u elaborate what u mean by this? my kitty terminal only shows a black screen but i can type fine in it. changing the .conf file for kitty isnt helping
